torchvision/transforms/transforms.py

```diff
@@ -753,21 +757,39 @@ def get_params(brightness, contrast, saturation, hue):
             Transform which randomly adjusts brightness, contrast and
             saturation in a random order.
         """
+        def _sample_from(value, name):
+            factor = None
+            if isinstance(value, numbers.Number) and value >= 0:
+                if name == 'hue':
+                    factor = random.uniform(-value, value)
+                else:
+                    factor = random.uniform(max(0, 1 - value), 1 + value)
+            elif isinstance(value, (tuple, list)) and len(value) == 2:
+                if (name == 'hue' and not (-0.5 <= value[0] <= value[1] <= 0.5)) or \
+                   (name != 'hue' and not (0 <= value[0] <= value[1])):
+                    raise ValueError('Invalid {} factor {}.'.format(name, value))
+                factor = random.uniform(value[0], value[1])
+            else:
+                raise TypeError('{} should be non negative float or tuple of float (min, max).'.format(name))
+            return factor
+
         transforms = []
-        if brightness > 0:
-            brightness_factor = random.uniform(max(0, 1 - brightness), 1 + brightness)
+
+        brightness_factor = _sample_from(brightness, 'brightness')
+        if brightness_factor is not None:
             transforms.append(Lambda(lambda img: F.adjust_brightness(img, brightness_factor)))
```

Comment:
I have the impression that now we always append the transform, even if brightness is 0, which is a performance regression compared to the previous implementation.

Reply:
The current implementation doesn't append the transform when brightness is 0. See [Line 757](https://github.com/pytorch/vision/blob/master/torchvision/transforms/transforms.py#L757), so should we keep this or append the transform whatever the value is?

Reply:
What happens if brightness is set to zero in the current code? It doesn't change anything in the input, right?
In that case, why don't we just skip the transform, which will make the code a bit more efficient?